trio/ssl.py

```diff
@@ -4,5 +4,36 @@
 # temporaries, imports, etc. when implementing the module. So we put the
 # implementation in an underscored module, and then re-export the public parts
 # here.
-from ._ssl import *
-from ._ssl import __all__
+
+from ._ssl import (SSLStream, SSLListener, aclose_forcefully, ConflictDetector)
```

Comment:
This should just be `SSLStream` and `SSLListener`... the other ones were intentionally left out of `__all__` before :-)